package.json

```diff
@@ -51,8 +51,8 @@
     "devDependencies": {
         "@eslint/js": "9.17.0",
         "@types/node": "20.12.2",
-        "@typescript-eslint/eslint-plugin": "8.19.1",
-        "@typescript-eslint/parser": "8.19.1",
+        "@typescript-eslint/eslint-plugin": "8.20.0",
```

Comment:
Bumped these to try to fix the build failure you can see in the commit history. I figured I might as well keep the upgrade around.